matchers/be_above.go

```diff
@@ -2,10 +2,13 @@ package matchers
 
 import "fmt"
 
+// BeAboveMatcher accepts a numerical value. It succeeds if the
```

Comment:
Really the value must be a float64 and not any numeric. This is something we might want to support though.